### ISSUE TITLE
write docstrings for Room.onStabilityChange, Room.adjustStability, Room.keenlyObserve, and Room.coolFire -- refactor the names of these functions after.

### ISSUE THREAD

**Issue by shaenr** [shaenr/aparentnode#5]
There are two methods with similar names that seem to be for a similar purpose but they are not and actually do different things. Somehow this mess was made, so you need to go through everywhere that either is called and figure out what is important from those methods and what can be removed. There should better names so its clear what these methods are doing.

**Comment by shaenr** [comment]
Docstrings added to `Room.adjustStability`

```js
        /** Room.onStabilityChange
	 * [does various checks that occur any time a method changes stability/network/fire/entropy (see coolFire and adjustStability); 
	 * Sets builder level, resets timer before decrease in stability, adjusts brightness of background.]
	 * @return {[void]}
	 */

	/** Room.adjustStability
	 * [Causes squat stability to increment or decrement based on the value of 'network'
	 * Is called on a timer determined by Room._ROOM_WARM_DELAY]
	 * @return {[void]}
	 */
```

When I wrote the issue I thought they were maybe duplicate functions called in different places in error. 
`onStabilityChange` is a helper that follows `adjustStability` -- or really, any function that adjust stability. I should make it so that only this one function is responsible for any changes, but they're also occuring in Room.`observeKeenly` where I just have to remember to follow up with `onStabilityChange` 

`adjustStability` is only called by the `setTimeout` regulated by _ROOM_WARM_DELAY
on the other hand, `observeKeenly` has to do with changing the stability based on having pressed the `Stay Sharp` button.

This makes me realize, the Stay Sharp button should be affecting "network", which in turn adjusts stability when the `adjustStability` gets called naturally. This would explain why the `'game.network.value'` seems to always be at zero, which I discovered with the new `Engine.logFromSM` function.

---

In conclusion, I need to follow by docstringing `coolFire`, `keenlyObserve` and other functions that affect 'network'

**Comment by shaenr** [comment]
the relevant methods, associated vars, and state manager locations have all been refactored to much more sensible names, though I have not merged refactor branch into main yet just to be safe. See [this commit](https://github.com/shaenr/aparentnode/commit/f8fd24201bffeb7dcfacbbb104a211fa3ebd7419)

I still need to write the docstrings on a few things to officially close issue but issue is pretty resolved.
